caffe2/core/blob_test.cc

```diff
@@ -538,12 +538,19 @@ TEST(TensorTest, Tensor64BitDimension) {
   EXPECT_EQ(tensor.ndim(), 1);
   EXPECT_EQ(tensor.dim(0), large_number);
   EXPECT_EQ(tensor.size(), large_number);
-  EXPECT_TRUE(tensor.mutable_data<char>() != nullptr);
   EXPECT_EQ(tensor.nbytes(), large_number * sizeof(char));
   EXPECT_EQ(tensor.itemsize(), sizeof(char));
-  // Try to go even larger, but this time we will not do mutable_data because we
-  // do not have a large enough memory.
-  tensor.Resize(large_number, 100);
+  try {
+    EXPECT_TRUE(tensor.mutable_data<char>() != nullptr);
+    // Try to go even larger, but this time we will not do mutable_data because we
+    // do not have a large enough memory.
+    tensor.Resize(large_number, 100);
+  } catch (const EnforceNotMet& e) {
+    string msg = e.what();
```

Comment:
As mentioned in the above comment: you should do a string search for aligned_malloc, and only when the exception is caused by aligned malloc should we ignore it.